azure-pipelines.release-vnext-nightly.yml

```diff
@@ -36,9 +36,8 @@ steps:
 
     # Bumps all v9 packages to a x.x.x-nightly.commitSha version and checks in change files
   - script: |
-      commitSha=$(git log -n 1 --pretty=format:"%h")
-      date=$(date +"%Y%m%d")
-      yarn nx workspace-generator version-bump --all --bumpType nightly --prereleaseTag "nightly${commitSha}${date}"
+      date=$(date +"%Y%m%d.%H%M")
```

Comment:
Can you add a comment with an example of how this format will turn out, like in the PR description?